### ISSUE TITLE
Use in CI like GitHub actions

### ISSUE THREAD

**Issue by twhitbeck** [cloudflare/wrangler2#24]
I'm trying to use wrangler2 to publish in a GitHub action. Previously, I was using https://github.com/cloudflare/wrangler-action and providing `apiToken`. Is there a way to provide this api token to `wrangler2`? Here's what I'm currently trying:

https://github.com/twhitbeck/shortr-cf/blob/main/.github/workflows/worker.yml

Action result:
https://github.com/twhitbeck/shortr-cf/runs/4228349780

also worth nothing that the action succeeded even though wrangler complains:

```
Timed out waiting for authorization code.
Did not login, quitting...
```

**Comment by threepointone** [comment]
Yes, we definitely want to support this flow. In fact, I think I'd got it working previously, and might have broken it. I'll fix this soon. 

To clarify, yes, we should read an api token from `CF_API_TOKEN`.

**Comment by twhitbeck** [comment]
[this](https://blog.cloudflare.com/cloudflare-pages-goes-full-stack/) changes the game for me here. I'll be switching to this and not using a GitHub action to deploy my worker anymore. 🎉

probably still worth addressing :)

**Comment by threepointone** [comment]
Yeah, full stack pages is awesome :)
